docs/DeveloperGuide.md

```diff
@@ -2583,81 +2583,109 @@ Expected: The GUI closes and the application exits.
 
 #### Adding an Exam: `addExam`
 
-##### Adding an Exam with Valid Data
+**Command:** `addExam`<br>
+**More information on usage:** <a href="UserGuide.md#addexam">Adding an Exam</a>
 
-1. Prerequisites: No exams in the address book.
+1. Adding an Exam with Valid Data
 
-2. Test case: `addExam n|Midterm s|100`<br>
-   Expected: New exam is added to the address book. Status message shows the exam added.
+   * **Prerequisites:**
+       * No exams in the address book.
+         <br><br>
+   * **Test case:** `addExam n|Midterm s|100`<br>
+     **Expected:** New exam is added to the address book. Status message shows the exam added.
+     <br><br>
+   * **Other test cases to try:** `addExam n|Final s|100`<br>
+     **Expected:** New exam is added to the address book. Status message shows the exam added.
+     <br><br>
 
-3. Other test cases to try: `addExam n|Final s|100`<br>
-   Expected: New exam is added to the address book. Status message shows the exam added.
+2. Adding an Exam that Already Exists
 
-##### Adding an Exam that Already Exists
+   * **Prerequisites:**
+       * An exam of name: Final, Score: 100 exists in the address book.
+       <br><br>
+   * **Test case:** `addExam n|Final s|100`<br>
+     **Expected:** Error message shown in the error report. No change in the address book.
+     <br><br>
 
-1. Prerequisites: An exam of name: Final, Score: 100 exists in the address book.
+3. Adding an Exam with Missing Fields
 
-2. Test case: `addExam n|Final s|100`<br>
-   Expected: Error message shown in the error report. No change in the address book.
-
-##### Adding an Exam with Missing Fields
-
-1. Pre-requisite: No exams in the address book.
-
-2. Test case: `addExam n|Final` (missing score)<br>
-     Expected: Error message shown in the error report. No change in the address book.
+   * **Prerequisites:** 
+       * No exams in the address book.
+         <br><br>
+   * **Test case (missing score):** `addExam n|Final`<br>
+     **Expected:** Error message shown in the error report. No change in the address book.
```

Comment:
Do we still refer to the app as Address Book? Maybe we should change it to "No change in Exams List"